README.md

```diff
@@ -1,4 +1,4 @@
-![CI](https://github.com/statropy/msp430F55xx_usb_uart_bridge/workflows/CI/badge.svg)
+![CI](https://github.com/jadonk/msp430F55xx_usb_uart_bridge/workflows/CI/badge.svg)
```

Comment:
The badge here should not change. Seems like the badge for your CI should be in a top-level README with all the submodules listed